### PR TITLE
docs(cloud): update sending emails section

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Comment by aeneasr** [comment]
Formatter is failing with unrelated files, I guess this is part of the PRs that are still open

File: docs/concepts/emails.mdx [hunk 1]
```diff
@@ -9,15 +9,98 @@ Per default, Ory Cloud has the SMTP email sending configured. Ory Cloud sends Em
...
+ory patch project {your-project-id} \
+  --replace '/services/identity/config/courier/smtp/connection_uri="smtp://username:password@server:port/"' \
+  --replace '/services/identity/config/courier/smtp/from_address="hello@example.org' \
+  --replace '/services/identity/config/courier/smtp/from_name="My Company"'
+```
+
+### SMTP Connection URI
+
+SMTP has six different security mechanisms. Most SMTP services today use Explicit StartTLS with trusted certificates - the item in
+the list below.
```
Comment (tomekpapiernik) [inline docs/concepts/emails.mdx:39]:
```suggestion
SMTP has six different security mechanisms. Most SMTP services today use Explicit StartTLS with trusted certificates.
```

Comment (tomekpapiernik) [inline docs/concepts/emails.mdx:39]:
```suggestion
SMTP has six different security mechanisms. Most SMTP services today use Explicit StartTLS with trusted certificates.
```

File: docs/concepts/emails.mdx [hunk 2]
```diff
@@ -9,15 +9,98 @@ Per default, Ory Cloud has the SMTP email sending configured. Ory Cloud sends Em
...
+
+1. **Recommended**: StartTLS with certificate trust verification. This is the most common option today:
+   ```
+   smtp://username:password@server:port/
+   ```
+1. StartTLS without certificates trust verification:
```
Comment (tomekpapiernik) [inline docs/concepts/emails.mdx:45]:
```suggestion
1. StartTLS without certificate trust verification:
```

File: docs/concepts/emails.mdx [hunk 3]
```diff
@@ -9,15 +9,98 @@ Per default, Ory Cloud has the SMTP email sending configured. Ory Cloud sends Em
...
+   ```
+   smtp://username:password@server:port/?skip_ssl_verify=true
+   ```
+1. Cleartext SMTP uses no encryption and is not secure. This option is often used in development environments:
+   ```
+   smtp://username:password@server:port/?disable_starttls=true
+   ```
+1. Implicit TLS with certificates trust verification:
```
Comment (tomekpapiernik) [inline docs/concepts/emails.mdx:53]:
```suggestion
1. Implicit TLS with certificate trust verification:
```

File: docs/concepts/emails.mdx [hunk 4]
```diff
@@ -9,15 +9,98 @@ Per default, Ory Cloud has the SMTP email sending configured. Ory Cloud sends Em
...
+   ```
+   smtps://username:password@server:port/
+   ```
+1. Implicit TLS without certificates trust verification:
```
Comment (tomekpapiernik) [inline docs/concepts/emails.mdx:57]:
```suggestion
1. Implicit TLS without certificate trust verification:
```

File: docs/concepts/emails.mdx [hunk 5]
```diff
@@ -9,15 +9,98 @@ Per default, Ory Cloud has the SMTP email sending configured. Ory Cloud sends Em
...
+
+# For example:
+# smtp://thetoken:thetoken@smtp.postmarkapp.com:587/
+```
+
+### Error Transparency
+
+In the future, we will improve error observability and transparency for this feature. If you have problems with email delivery
+over custom SMTP servers, please, for the time being, reach out to us on [Slack](https://slack.ory.sh) or
+[GitHub](https://github.com/ory/cloud) for assistance.
```
Comment (tomekpapiernik) [inline docs/concepts/emails.mdx:106]:
```suggestion
### Troubleshooting

If you have problems setting up email delivery or you experience errors or bugs, reach out to the Ory team directly:

- Send us a message on [Slack](https://slack.ory.sh)
- Create an issue on [GitHub](https://github.com/ory/cloud)
```

Just a reminder - promising things in the documentation isn't a good practice. The docs should focus on the current state of things. "In the future" can mean different things to different people, including us. Therefore I extracted the important message from this paragraph and gave it a more suitable (IMO) heading. ;)

**Review by tomekpapiernik — CHANGES_REQUESTED** [review]

File: docs/concepts/emails.mdx (hunk 5 above)
Comment (aeneasr) [reply docs/concepts/emails.mdx:105]: Makes sense, the thinking here being that this state is really not acceptable (errors should be observable for our customers) and that we know that -> fixing it.

Mabye we can reference the issue? https://github.com/ory/cloud/issues/117

Comment (tomekpapiernik) [reply docs/concepts/emails.mdx:105]: @aeneasr I edited the suggestion to include a note that references the RPC issue. 👍🏻 or 👎🏻 ?

**Comment by aeneasr** [comment]
Thank you tom!

**Review by tomekpapiernik — APPROVED** [review]

**Pull request merged**